src/main/java/com/linkedin/kmf/services/configs/TopicManagementServiceConfig.java

```diff
@@ -76,7 +77,7 @@
               ConfigDef.Importance.LOW, REBALANCE_INTERVAL_MS_DOC)
       .define(PARTITIONS_TO_BROKER_RATIO_CONFIG,
               ConfigDef.Type.DOUBLE,
-              2.0,
+              PARTITIONS_TO_BROKER_RATIO_DEFAULT,
```

Comment:
@smccauliff I am not if I understand you correctly. Are you suggesting that we should make all defaults a variable? Since this default value only needs to be specified in the config class once, other classes won't need to reference default value explicitly. Then what is the purpose of making assigning default value to a variable?